packages/web/src/js/utils/ScrollControl.ts

```diff
@@ -0,0 +1,43 @@
+import { SelectorEngine } from '../dom';
+
+const DIALOG_OPEN_SELECTOR = 'dialog[open]';
+const SCROLLING_DISABLED_CLASSNAME = 'is-scrolling-disabled';
+
+class ScrollControl {
+  element: HTMLElement;
+  offsetY: number;
+
+  constructor() {
+    this.element = document.body;
+    this.offsetY = 0;
+  }
+
+  disableScroll() {
+    // Browser scrollbar width, may be 0 on some devices, but solves offset when is not 0.
+    const scrollBarWidth = this.element.offsetWidth - this.element.clientWidth;
```

Comment:
tested in windows, both of these are equal. We could either use window.innerWidth instead of the offsetWidth or create temp element to measure it.

Reply:
done